stack/app/ecs.tf

```
@@ -49,15 +49,16 @@ module "ecs" {
   grant_read_access_to_sqs_arns  = var.grant_read_access_to_sqs_arns
   grant_write_access_to_sqs_arns = var.grant_write_access_to_sqs_arns
 
-  custom_policies       = var.ecs_custom_policies
-  enable_dashboard      = var.cloudwatch_dashboard_view == "simple"
-  enable_xray           = var.enable_xray
-  autoscale_params      = var.autoscale_params
-  autoscale_metrics_map = var.autoscale_metrics_map
+  custom_policies                   = var.ecs_custom_policies
+  enable_dashboard                  = var.enable_cloudwatch_dashboard == true && var.cloudwatch_dashboard_view == "simple"
```

```suggestion
  enable_dashboard                  = var.enable_cloudwatch_dashboard && var.cloudwatch_dashboard_view == "simple"
```

do we need `== true` for booleans?